mariadb/repo.sls

```diff
@@ -1,3 +1,4 @@
+{%- set os_family = salt['grains.get']('os_family').lower() %}
 {%- set lsb_codename = salt['grains.get']('lsb_distrib_codename') %}
 {%- set stable_version = '10.1' %}
```

Comment:
The current stable version is 10.2 at the moment, but since its a default value I'm not sure if we can/should just change it.